apps/ledger-live-desktop/tests/models/SettingsPage.ts

```diff
@@ -25,16 +24,16 @@ export class SettingsPage {
     this.experimentalDevModeToggle = page.locator("data-test-id=MANAGER_DEV_MODE-button");
     this.carouselSwitchButton = page.locator("data-test-id=settings-carousel-switch-button");
     this.counterValueSelector = page.locator(
-      "//*[@data-test-id='setting-countervalue-dropDown']/div[2]/div/div",
+      "[data-test-id='setting-countervalue-dropDown'] .select__value-container",
     );
     this.counterValueSearchBar = page.locator('[placeholder="Search"]');
-    this.counterValueropdownChoiceEuro = page.locator(
-      "#react-select-2-option-15 >> text=Euro - EUR",
-    );
+    this.counterValueropdownChoiceEuro = page.locator(".select__option");
```

Comment:
This seems like a very generic locator, and not related to the choice of 'Euro'. Is there just one option?

Reply:
yes, cause in the test, i filled the search input with 'euro' one step before, so it only have choice of 'Euro' as selection

Reply:
Ok sounds good 👍